spec/index.md

```diff
@@ -504,13 +498,25 @@ Whilst strictly not an entity, the concept of an "account" from which money is s
 
 ### Location
 
-There is a frequent desire to label items with location, usually by attaching geographic codes for a region or area. This allows the spending or revenue to  be analysed by region or area. This geographic information can be introduced directly by classifying the item with a code, or, more frequently indirectly by associating a geographic code to e.g. an entity. For example, by labelling a supplier with their location one can then associate a spend with that supplier as spending in that location.
+There is a frequent desire to label items with location, usually by attaching geographic codes for a region or area. This allows the spending or revenue to be analysed by region or area. This geographic information can be introduced directly by classifying the item with a code, or, more frequently indirectly by associating a geographic code to e.g. an entity. For example, by labelling a supplier with their location one can then associate a spend with that supplier as spending in that location.
 
 We RECOMMEND using a `location` dimension though attributes may also be applied directly onto another object (e.g. an entity). Here are attributes that `MAY` be applied either directly to an item or to an entity or other object associated to an item.
 
 * `code`: The internal or local geographicCode id based for the geographical region
 * `title`: Name or title of the geographical region targeted by the budget item
-* `codeList`: the geo codelist from which the geocode is drawn 
+* `codeList`: The geo codelist from which the geocode is drawn 
+* `countryCode`: A valid 2-digit ISO country code (ISO 3166-1 alpha-2). Note: if the entire dataset relates to one country, it is recommended to use the `constant` attribute to specify this.  For example:
+
+```
+"location": {
+  "dimensionType": "location",
+  "attributes": {
+    "countryCode": {
+      "constant": "GB"
+    }
+  }
+}
+```
 
 Note when applying these as attributes directly on an object we suggest prefixing each value with `geo` so you would have `geoCode` rather than `code` etc.
```

Comment:
Ah, is supposed to mean, when applying location attributes to a *different* dimension?

If yes, it is really hard to grok this: you are actually talking about two use cases:

1. A location dimension
2. Adding location attributes to other dimensions.